app/assets/stylesheets/components/common/_news_signup_sticky.scss

```diff
@@ -0,0 +1,198 @@
+// Newsletter signup sticky footer component
+//
+// Styleguide Newsletter Signup Sticky
+
+.news-signup-sticky {
+  @extend %stripe-bg;
+  @include clearfix;
+  background-color: $color-yellow-light;
+  margin: $baseline-unit*2 0 0 0;
+  text-align: center;
+  position: relative;
+
+  .button--newsletter {
+    vertical-align: top;
+  }
+}
+
+.news-signup-sticky--positioned {
+  visibility: hidden;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  z-index: 100;
+  max-height: 0;
+  transition: 2s cubic-bezier(0, 1.05, 0, 1);
+}
+
+.news-signup-sticky--visible {
+  visibility: visible;
+  max-height: 1000px;
+  transition: max-height 1s;
+}
+
+.news-signup-sticky__intro {
+  @include body(16,20);
+  color: $color-true-black;
+  font-weight: 500;
+  margin: 0 auto $baseline-unit*3 auto;
+  padding-top: $baseline-unit*3;
+  max-width: 240px;
+  position: relative;
+
+  @include respond-to($mq-m) {
+    @include body(17,19);
+    max-width: 400px;
+  }
+}
+
+.news-signup-sticky__intro--emphasize {
+  font-weight: 700;
+  margin-top: $baseline-unit*3;
+  padding-top: 0;
+
+  @include respond-to($mq-m) {
+    @include body(16,18);
+  }
+
+  .is-errored & {
+    margin-top: 0;
+    padding-top: $baseline-unit*3;
+  }
+}
+
+.news-signup-sticky__intro--success {
+  margin: $baseline-unit*5 auto $baseline-unit*3 auto;
+}
+
+.news-signup-sticky__error {
+  display: none;
+
+  &.news-signup-sticky__error--display {
+    display: block;
+  }
+
+  @include respond-to($mq-m) {
+    margin: $baseline-unit*3 auto $baseline-unit*2 auto;
+    padding: 0 0 $baseline-unit 0;
+    max-width: 400px;
+  }
+}
+
+.news-signup-sticky__text {
+  display: none;
+
+  &.news-signup-sticky__text--display {
+    display: block;
+  }
+}
+
+.news-signup-sticky__submitform {
+  .button--done {
+    margin: $baseline-unit*3 0 $baseline-unit*5 0;
+    min-width: 200px;
+  }
+}
+
+.news-signup-sticky__inner {
+  @include respond-to($mq-m) {
+    &:after {
+      background: image-url('newsletter_signup_image.png') no-repeat top right;
+      background-size: 140px 132px;
+      content: " ";
+      height: 132px;
+      left: 0;
+      position: absolute;
+      top: -5px;
+      width: 20%;
+    }
+  }
+
+  @include respond-to($mq-l) {
+    &:after {
+      width: 25%;
+    }
+  }
+
+  @include respond-to($mq-xl) {
+    &:after {
+      width: 32%;
+    }
+  }
+}
+
+.news-signup-sticky__email-label {
+  @include visually-hidden;
+}
+
+input[type="email"].news-signup-sticky__email-input {
+  @include body(14, 14);
+  border-color: $color-newsletter;
+  border-radius: 4px;
+  margin: $baseline-unit;
+  max-width: 188px;
+  margin: 0 auto $baseline-unit*1.5 auto;
+  display: block;
+  padding: $baseline-unit*3 $baseline-unit;
+
+  &:focus {
+    outline: 2px solid $color-newsletter-active;
+  }
+
+  @include respond-to($mq-m) {
+    display: inline-block;
+    margin: 0 $baseline-unit*2;
+    position: relative;
+    top: 1px;
+  }
+
+  &::-webkit-input-placeholder {
```

Comment:
Can you see what autoprefixer does with this? There are other prefixes I believe.